### PR TITLE
[Kotlin] Add Kotlin track maintainers

### PR DESCRIPTION
**Pull request by dector**
Only those maintainers added who responded [here](https://github.com/exercism/kotlin/issues/406) in the same order.

### CODE REVIEW

File: languages/kotlin/maintainers.md [hunk 1]
```diff
@@ -8,4 +8,7 @@ TODO: add senior maintainers
 
 ## Contributing Maintainers
 
-TODO: add contributing maintainers
+  - Elena Parovyshnaia (Slack: `@Elena Parovyshnaia`, Github: [eparovyshnaya](https://github.com/eparovyshnaya))
+  - Uzi Landsmann (Slack: `@uzilan`, Github: [uzilan](https://github.com/uzilan))
+  - Mike Gehard (Slack: `@mikegehard`, Github: [mikegehard](https://github.com/mikegehard))
+  - Denys M (Slack: `@dector`, Github: [dector](https://github.com/dector))
```
Comment (aimorris) [inline languages/kotlin/maintainers.md]: Replace Github with GitHub for consistency in all maintainers.md files

Comment (uzilan) [reply languages/kotlin/maintainers.md]: Done

**Comment by uzilan** [comment]
Why is this still blocked? I resolved @aimorris request

**Comment by NobbZ** [comment]
> Why is this still blocked? I resolved @aimorris request

They need to approve, just resolving the discussion does not fullfil this requirement.

File: languages/kotlin/maintainers.md (hunk 1 above)
Comment (dector) [reply languages/kotlin/maintainers.md]: My bad. I thought I copied it from C# template.

**Pull request merged**